terraform/environments/maat/api-ecs.tf

```diff
@@ -266,6 +266,47 @@ resource "aws_cloudwatch_metric_alarm" "maat_api_low_cpu_service_alarm" {
   comparison_operator = "LessThanThreshold"
   alarm_actions       = [aws_appautoscaling_policy.maat_api_scaling_down_policy.arn]
 
+  dimensions = {
+    ClusterName = aws_ecs_cluster.app_ecs_cluster.name
+    ServiceName = aws_ecs_service.maat_api_ecs_service.name
+  }
+}
+
+## Memory Based Scaling
+resource "aws_cloudwatch_metric_alarm" "maat_api_high_memory_service_alarm" {
+  alarm_name          = "${local.application_name}-api-high-memory-service-alarm"
+  alarm_description   = "MemoryUtlization exceeding threshold. Triggers scale up"
+  actions_enabled     = true
+  namespace           = "AWS/ECS"
+  metric_name         = "MemoryUtilization"
+  statistic           = "Average"
+  period              = 60
+  evaluation_periods  = 3
+  threshold           = 70
```

Comment:
For the alarm thresholds on scaling, could you make them all into variables please, so they can be adjusted per environment? The CPU ones would need changing as well.